spk/minio/src/wizard/upgrade_uifile

```diff
@@ -0,0 +1,6 @@
+[{
+    "step_title": "Attention! DSM6 Permissions",
+    "items": [{
+        "desc": "Permissions for this package are handled by the <b>'sc-minio'</b> group. <br>Using File Station, add this group to every folder minio should be allowed to access.<br>Please read <a <a target=\"_blank\" href=\"https://github.com/SynoCommunity/spksrc/wiki/Permission-Management\">Permission Management</a> for details."
```

Comment:
a bit weird that you leave a choice to set a custom group name above but refer to the specific group `sc-minio` here

Reply:
Fixed in latest commit.